File: problems/problems_3784/solution.py
from collections import defaultdict

import solution
from typing import *


class Solution(solution.Solution):
    def solve(self, test_input=None):
        return self.minCost(*test_input)

    def minCost(self, s: str, cost: List[int]) -> int:
        counts = defaultdict(int)
        sm = 0
        for c, ct in zip(s, cost):
            counts[c] += ct
            sm += ct
        return sm - max(counts.values())

File: problems/problems_3784/Solution.java
package problems.problems_3784;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public long minCost(String s, int[] cost) {
        
    }

    @Override
    public Object solve(String[] inputJsonValues) {
        String s = jsonStringToString(inputJsonValues[0]);
		int[] cost = jsonArrayToIntArray(inputJsonValues[1]);
        return JSON.toJSON(minCost(s, cost));
    }
}
